app/Filament/Resources/DatabaseHostResource/Pages/ListDatabaseHosts.php

```diff
@@ -31,14 +31,17 @@ public function table(Table $table): Table
                 TextColumn::make('port')
                     ->sortable(),
                 TextColumn::make('username')
+                    ->icon('tabler-user')
```

Comment:
Wouldn't adding a user icon suggest that it's a panel user?

Reply:
Good point